common/lib/kontena-logging.rb

```diff
@@ -0,0 +1,3 @@
+module Kontena
+  require_relative './kontena/logging'
```

Comment:
AFAIK besides ruby's requires doesn't really allow for requiring specific sub-parts of packages in general, you generally have a single top-level require.

But I don't want to make `kontena-common.rb` require every possible thing either... so several separate top-level requires like this instead?